APMrover2/Parameters.h

```diff
@@ -22,6 +22,9 @@ class Parameters {
         //
         k_param_format_version = 0,
         k_param_software_type,
+#if HAL_WITH_UAVCAN
```

Comment:
This shouldn't be protected because it reserves the parameter index that should always be the same, whether HAL has UAVCAN or not. Same for all vehicles.